### PR TITLE
feat(youtube-player): Add proxy methods for calling the underlying pl…

### PR DESCRIPTION
**Pull request by YourDeveloperFriend**
…ayer

### CODE REVIEW

File: src/youtube-player/youtube-player.ts [hunk 1]
```diff
@@ -115,6 +115,147 @@ export class YouTubePlayer implements AfterViewInit, OnDestroy {
     this._player.destroy();
     this._destroyed.emit();
   }
+
+  /** Proxied methods. */
```
Comment (jelbourn) [inline src/youtube-player/youtube-player.ts]: Could you add a JsDoc for each of these? It's fine if they're all just the same link to the full reference, e.g.
```ts
See https://developers.google.com/youtube/iframe_api_reference#Functions
```

**Comment by YourDeveloperFriend** [comment]
PR is ready for review.

**Comment by jelbourn** [comment]
Looks like a build error on CI

**Comment by YourDeveloperFriend** [comment]
PR is ready for review.

**Pull request merged**